tools/deployment/make_linux_package.sh

```diff
@@ -136,6 +137,15 @@ function main() {
     cp $INITD_SRC $INSTALL_PREFIX$INITD_DST
   fi
 
+  if [[ $DISTRO = "xenial" ]]; then
```

Comment:
That would do it, the intention of my last solution was to encapsulate all of this into a single if statement, but two effective statements is more efficient.